resources/standards-data/ldml-keyboards/techpreview/dtd/ldmlKeyboard.dtd

```diff
@@ -13,7 +13,7 @@ This DTD is a work in progress.
 
 Please see CLDR-15034 for the latest information. -->
 
-<!ELEMENT keyboard ( import*, locales?, version?, info?, names, settings?, vkeyMaps*, displayMap*, keys, layerMaps*, transforms*, reorders?, backspaces?, special* ) >
+<!ELEMENT keyboard ( import*, locales?, version?, info?, names, settings?, vkeys*, displays*, keys?, layers*, transforms*, reorders?, special* ) >
```

Comment:
```suggestion
<!ELEMENT keyboard ( import*, locales?, version?, info?, names, settings?, vkeys*, displays?, keys?, layers*, transforms*, reorders?, special* ) >
```

Matches https://github.com/unicode-org/cldr/pull/2469

Reply:
pulled in via #7479

Reply:
done